UI/data/locale/en-US.ini

```diff
@@ -833,7 +830,7 @@ Basic.Settings.Audio.PeakMeterType="Peak Meter Type"
 Basic.Settings.Audio.PeakMeterType.SamplePeak="Sample Peak"
 Basic.Settings.Audio.PeakMeterType.TruePeak="True Peak (Higher CPU usage)"
 Basic.Settings.Audio.MultiChannelWarning.Enabled="WARNING: Surround sound audio is enabled."
-Basic.Settings.Audio.MultichannelWarning="If streaming, check to see if your streaming service supports both surround sound ingest and surround sound playback. Facebook 360 Live, Mixer RTMP, Smashcast are examples where surround sound is fully supported. Although Facebook Live and YouTube Live both accept surround ingest, Facebook Live downmixes to stereo, and YouTube Live plays only two channels.\n\nOBS audio filters are compatible with surround sound, though VST plugin support isn't guaranteed."
+Basic.Settings.Audio.MultichannelWarning="If streaming, check to see if your streaming service supports both surround sound ingest and surround sound playback. Facebook 360 Live, Smashcast are examples where surround sound is fully supported. Although Facebook Live and YouTube Live both accept surround ingest, Facebook Live downmixes to stereo, and YouTube Live plays only two channels.\n\nOBS audio filters are compatible with surround sound, though VST plugin support isn't guaranteed."
```

Comment:
Should probably read "Facebook 360 Live and Smashcast"

Reply:
I'd probably remove the Smashcast mention entirely since that site is basically dead.

Reply:
Either way. Both the previous iteration of this sentence and the current iteration are both grammatically incorrect.